src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -366,7 +366,7 @@ public FormValidation doCheckServerId(@AncestorInPath Item context,
 
         @Override
         @POST
-        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item context,
+        public HttpResponse doFillCredentialsIdItems(@AncestorInPath Item context,
```

Comment:
Again, same change, seems unrelated to the task at hand? I would have thought it would be better to stay as a `ListBoxModel` as that is what the FE needs. While this works, I don't understand why we've changed it?

I won't comment on all of them, but there are a few of these scattered across the changeset.